deploy-with-init-params.md

```diff
@@ -39,111 +37,95 @@ forgae node
+Creating forgae-init_node1_1 ...
+Creating forgae-init_proxy_1 ...
+Creating forgae-init_node2_1 ...
+..............
+
+===== Node was successfully started =====
+===== Funding default wallets =====
+
+....
 ===== Default wallets was successfully funded! =====
 ```
 
-#### Do not forget to stop it later once you are done developing
+Do not forget to stop it once you are done developing
 
 ```
 forgae node --stop
 ```
 
 ## Step 1: Update your ExampleContract.aes
 
-Let's add some state and init parameters to our example contract which can be found at **contracts/ExampleContract.aes**
+Lets add some state and init parameters to our example contract which can be found at **contracts/ExampleContract.aes**
 
 ```
 contract ExampleContract =
-  record state =
-    { savedNumber : int }
 
-  public stateful function init(num : int) =
-    { savedNumber = num }
+  record state = { saved_string : string }
+
+  entrypoint init() = { saved_string = "aeternity" }
 
-  public function savedNumber() : int = state.savedNumber
+  public entrypoint get_string() : string =
+	  state.saved_string
+
+  public stateful entrypoint register_string(word : string) =
+	  put(state { saved_string = word})
 ```
 
-As you can see the contract now has a state variable `savedNumber` of type int. The initial value will be passed by the init function. We've also added a read function for this value.
+As you can see the contract now has a state variable `saved_string` of type `string`. The initial value will be passed by the init function. We've also added a read function `get_string` for this value and a write function `register_string` to write and change the value in `saved_string`.
 
 Run forgae compile to verify that your contract compiles successfully
 
 ```
 forgae compile
 ```
 
-#### Output of `forgae compile`:
-
-```
-===== Compiling contracts =====
-
-Contract '/home/justin/forgae-init/contracts/ExampleContract.aes has been successfully compiled'
-Contract bytecode: [contract bytecode]
-```
-
-## Step 2: Update your deploy.js
+## Step 2. Update your deploy.js
 
-Let's add some parameters to our example deploy script which can be found at **deployment/deploy.js**. The parameters of the init functions are always passed as an array. Here is how our new deploy script looks like:
+Lets add some parameters to our example deploy script which can be found at **deployment/deploy.js**. The parameters of the init functions are always passed as tuple. Here is how our new deploy script looks like
 
 ```
 const Deployer = require('forgae-lib').Deployer;
 
 const deploy = async (network, privateKey, compiler) => {
     let deployer = new Deployer(network, privateKey, compiler)
 
-	let contract = await deployer.deploy("./contracts/ExampleContract.aes", [42])
+    let contract = await deployer.deploy("./contracts/ExampleContract.aes")
+
+    // Getting savedString value in our ExampleContract
+    let get_string = await contract.call('get_string')
+    console.log(get_string.value)
```

Comment:
with current forgae, syntax is `await contract.decode()`, isn't it? and with the latest js-sdk is `decodedValue` ?